actors/builtin/miner/miner_actor.go

```diff
@@ -704,6 +704,7 @@ func (a Actor) ConfirmSectorProofsValid(rt Runtime, params *builtin.ConfirmSecto
 			activation := rt.CurrEpoch()
 			duration := precommit.Info.Expiration - activation
 			power := QAPowerForWeight(info.SectorSize, duration, precommit.DealWeight, precommit.VerifiedDealWeight)
+			dayReward := ExpectedDayRewardForPower(epochReward, pwrTotal.QualityAdjPower, power)
```

Comment:
Beware conceptual conflict with #762 (I think the type checker will see it).

Reply:
Should just be plumbing but I can be around to reduce rebase pain.